src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -39,16 +39,19 @@ def test_get_error(session):
         assert e.description.find('Attribute is read-only.') != -1
 
 
-'''
-TODO: (Jaleel) Dependent on PR#439 and Issue $428,#371 : Error -1074134972  (Hex 0xBFFA0044) Channel or or repeated capability name required
 def test_config_aperture_time(session):
-    session.configure_aperture_time(2, nidcpower.ApertureTimeUnits.POWER_LINE_CYCLES)
-    session.configure_aperture_time(0.01666, nidcpower.ApertureTimeUnits.SECONDS)
-    assert session.aperture_time_units == nidcpower.ApertureTimeUnits.SECONDS
-    assert session.aperture_time == 0.01666
+    with nidcpower.Session('FakeDevice', '0', False, 'Simulate=1, DriverSetup=Model:4143; BoardType:PXIe') as session:
+        session.configure_aperture_time(2, nidcpower.ApertureTimeUnits.POWER_LINE_CYCLES)
+        session.configure_aperture_time(0.01666, nidcpower.ApertureTimeUnits.SECONDS)
```

Comment:
Why do we call the same function twice? Should the first one be removed?

Reply:
Its to confirm the attributes are set properly by calling twice with different parameter and validating latest one is set properly

Reply:
A better test might be to do the following, where we assert against defaults. Change to non-default, then assert that it changed:

assert session.aperture_time_units == nidcpower.ApertureTimeUnits.POWER_LINE_CYCLES (make this match the default)
assert session.aperture_time == 2 (make this match the default)
session.configure_aperture_time(0.01666, nidcpower.ApertureTimeUnits.SECONDS)
assert session.aperture_time_units == nidcpower.ApertureTimeUnits.SECONDS
aperture_time = session.aperture_time
expected_aperture_time = 0.01666
....

Reply:
fixed